### DISCUSSION TITLE
Add Sitemap Index to Docs

### DISCUSSION THREAD

**Discussion by benhaynes** [directus/directus#9263] [Feature Requests]
We need to get better SEO on our Docs, especially for when we restructure things (as we did recently). Perhaps this repo can help us auto generate a sitemap index file from our Docs structure. Note: this only applies to the PUBLIC docs website, not in-app.

https://github.com/ekoeryanto/vuepress-plugin-sitemap

**Comment by SalahAdDin** [comment]
> We need to get better SEO on our Docs, especially for when we restructure things (as we did recently). Perhaps this repo can help us auto generate a sitemap index file from our Docs structure. Note: this only applies to the PUBLIC docs website, not in-app.
> 
> https://github.com/ekoeryanto/vuepress-plugin-sitemap

it seems there is no plugin for it yet.

**Comment by br41nslug** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._

**Reply by SalahAdDin** [reply]
Come on! This is something basic even the competence, Strapi, has.

**Reply by br41nslug** [reply]
This feature request is outdated and old enough it did not show up within the 1000 discussion limit on github before 😂 Feel free to open a new feature request or issue for this @SalahAdDin
